meson.build

```
@@ -432,6 +432,10 @@ if librsvg_found
     external_deps += librsvg_dep
     external_deps += cairo_dep
     cfg_var.set('HAVE_RSVG', '1')
+    # rsvg_handle_set_stylesheet added in librsvg 2.48.0
+    if cc.has_function('rsvg_handle_set_stylesheet', dependencies: librsvg_dep)
+        cfg_var.set('HAVE_RSVG_HANDLE_SET_STYLESHEET', '1')
+    endif
```

After rebasing, this can be simplified to:
```suggestion
    cfg_var.set('HAVE_RSVG_HANDLE_SET_STYLESHEET', cc.has_function('rsvg_handle_set_stylesheet', dependencies: librsvg_dep))
```